### ISSUE TITLE
TCGA Lancet data has only "calls" with t_alt_count = 0 or NA

### ISSUE THREAD

**Issue by cansavvy** [AlexsLemonade/OpenPBTA-analysis#512]
#### What data file(s) does this issue pertain to?
The TCGA Lancet data : `pbta-tcga-snv-lancet.vep.maf.gz` of version 14

#### What release are you using?

v14

#### Put your question or report your issue here.
I was confused because Lancet's data for TCGA was not agreeing at all with Mutect or Strelka. 
I looked into the VAF distributions and saw Lancet was all zeroes, because `t_alt_count` is only zeroes and NAs. `t_alt_count`'s  of 0 shouldn't be calls. Did something happen with a filtering step? It looks like it may have been filtered by `n_alt_count` > 0 ? 
```
lancet <- data.table::fread("../../data/pbta-tcga-snv-lancet.vep.maf.gz", data.table = FALSE)
summary(lancet$t_alt_count)

  Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA's 
      0       0       0       0       0       0     622 
```
As a positive control, and by contrast, Mutect and Strelka show no 0's for `t_alt_count`
```
mutect <- data.table::fread("../../data/pbta-tcga-snv-mutect2.vep.maf.gz", data.table = FALSE)
summary(mutect$t_alt_count)
Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
   2.00    4.00    8.00   19.09   21.00 3528.00 
```

**Comment by jharenza** [comment]
Hi @cansavvy - thanks for noticing this - was this also an issue in V13? I am assuming so, since this file should not have changed. I am cc-ing @tkoganti, @migbro, and @yuankunzhu for this.

**Comment by jaclyn-taroni** [comment]
I can confirm that it was also an issue for v13:
```r
> lancet <- data.table::fread("data/release-v13-20200116/pbta-tcga-snv-lancet.vep.maf.gz", data.table = FALSE)
> summary(lancet$t_alt_count)
   Min. 1st Qu.  Median    Mean 3rd Qu.    Max.    NA's 
      0       0       0       0       0       0     622 
```

**Comment by jharenza** [comment]
Quick spot check on individual MAFs - seems they are all like this, so not a merge/filter issue (no filtering for these). I asked those above to check out the run, as it appears something went wrong there.

**Comment by jharenza** [comment]
@cansavvy and @jaclyn-taroni it looks like for some reason, all of the lancet tasks were run with the tumor/normal inputs swapped, and the correct IDs were used in VEP/VCF2MAF, and that is why the strange output. @migbro is queueing these up to rerun tonight and I will ask @tkoganti to pick up in the morning. The strelka2/mutect2 MAF runs were run correctly. Thank you again for finding this!

**Comment by jharenza** [comment]
closed via #507
